### PR TITLE
HPCC-25560  update helm/examples/local README.md for  DD/Hyper-V

### PR DESCRIPTION
**Pull request by xwang2713**
Signed-off-by: Xiaoming Wang <xiaoming.wang@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Review by g-pan — COMMENTED** [review]
I approve

**Comment by richardkchapman** [comment]
Typo in commit message

**Comment by xwang2713** [comment]
Correct PR title: Hypber-V -> Hyper-V

**Comment by xwang2713** [comment]
Need update

**Comment by xwang2713** [comment]
Jim, please review the change for FILE-SHARING. Do we still add more information for OSX?

File: helm/examples/local/README.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ Examples of use:
   helm install localfile examples/local/hpcc-localfile --set common.hostpath=/User/myuser/hpccdata
```
Comment (JamesDeFabia) [inline helm/examples/local/README.md:27]: it should be: 
--set common.hostpath=/Users/myuser/hpccdata

File: helm/examples/local/README.md [hunk 2]
```diff
@@ -27,6 +27,14 @@ Examples of use:
...
   helm install mycluster hpcc/ --set global.image.version=latest -f examples/local/values-localfile.yaml
 
+#### Docker desktop (using hyper-v):
+
+  With a host directory of c:\hpccdata
+  It should be added to Docker Desktop Settings->Resources->FILE SHARING
+
+  helm install localfile examples/local/hpcc-localfile --set common.hostpath=/c/hpccdata
+  helm install mycluster hpcc/ --set global.image.version=latest -f examples/local/values-localfile.yaml
+
```
Comment (JamesDeFabia) [inline helm/examples/local/README.md:37]: In addition, you must add the folder in Docker Desktop's settings for Resources>File Sharing.

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Two comments inline

File: helm/examples/local/README.md (hunk 2 above)
Comment (xwang2713) [reply helm/examples/local/README.md:37]: changed

**Comment by xwang2713** [comment]
Jim, corrected and back to you. Thanks

File: helm/examples/local/README.md [hunk 3]
```diff
@@ -17,15 +17,23 @@ Examples of use:
...
 
-  With a host directory of /User/myuser/hpccdata
+  With a host directory of /Users/myuser/hpccdata
+
+    helm install localfile examples/local/hpcc-localfile --set common.hostpath=/Users/myuser/hpccdata
+    helm install mycluster hpcc/ --set global.image.version=latest -f examples/local/values-localfile.yaml
+
+#### Docker desktop (using hyper-v):
+
+  With a host directory of c:\hpccdata<br/>
```
Comment (JamesDeFabia) [inline helm/examples/local/README.md:32]: Is this  `<br>` needed here?

File: helm/examples/local/README.md [hunk 4]
```diff
@@ -41,7 +49,7 @@ Examples of use:
 
     minikube mount /home/myuser/hpccdata:/mnt/hpccdata --gid=1000 --uid=999
 
-    helm install localfile examples/local/hpcc-localfile --set common.hostpath=/mnt/hpccdata
+    helm install localfile examples/local/hpcc-localfile --set common.hostpath=/mnt/hpccdata<br/>
```
Comment (JamesDeFabia) [inline helm/examples/local/README.md:52]: Is this  `<br>` needed here?

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
There are a couple of  `<br>`  tags in the text, not sure they are needed.

File: helm/examples/local/README.md (hunk 4 above)
Comment (xwang2713) [reply helm/examples/local/README.md]: Yes, should remove it

**Review by JamesDeFabia — APPROVED** [review]
looks good

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Since this was approved, we made some changes. Please do not merge yet

**Comment by xwang2713** [comment]
Jim back to you

**Comment by xwang2713** [comment]
sync with candidate-8.0.x

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Line #12 should be
The chart assumes that the dalistorage, queries, sasha, and hpcc-data subdirectories already exist.

**Comment by xwang2713** [comment]
Update line 12.

**Review by JamesDeFabia — APPROVED** [review]
Looks good

**Pull request merged**